tests/commons/web/test_parse_headers.py

```
@@ -0,0 +1,70 @@
+import pytest
+
+from sema.commons.web import parse_header
+
+
+@pytest.mark.parametrize(
+    "mode, content, expected_main, expected_params",
```

Ruff (0.8.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)